helpers/mattermost.ts

```diff
@@ -111,14 +126,39 @@ export const publicChannelMessage: PublicChannelMessage = async (
   }
 }
 
-export const getUserByEmail = async (email: string): Promise<string> => {
+export const getUserByEmail = async (email: string): Promise<UserInfo> => {
   try {
     const user = await fetch(`${chatServiceUrl}/users/email/${email}`, {
       headers
     })
-    const { username } = await user.json()
-    return username
+    return await user.json()
   } catch (error) {
     throw new Error(error)
   }
 }
+
+export const getChatUserById = _.memoize(
+  async (id: string): Promise<UserInfo> => {
+    try {
+      const response = await fetch(`${chatServiceUrl}/users/${id}`, { headers })
+      return await response.json()
```

Comment:
you could do just `return response.json()` right?

Reply:
Right! 👍